DataConnectors/GoogleWorkspaceReports/GWorkspaceReports_API_FunctionApp.json

```diff
@@ -0,0 +1,211 @@
+	"graphQueries": [{
+			"metricName": "Total data received",
+			"legend": "GWorkspace_ReportsAPI_admin_CL",
+			"baseQuery": "GWorkspace_ReportsAPI_admin_CL"
+		},
+		{
+			"metricName": "Total data received",
+			"legend": "GWorkspace_ReportsAPI_calendar_CL",
+			"baseQuery": "GWorkspace_ReportsAPI_calendar_CL"
+		},
+		{
+			"metricName": "Total data received",
+			"legend": "GWorkspace_ReportsAPI_drive_CL",
+			"baseQuery": "GWorkspace_ReportsAPI_drive_CL"
+		},
+		{
+			"metricName": "Total data received",
+			"legend": "GWorkspace_ReportsAPI_login_CL",
+			"baseQuery": "GWorkspace_ReportsAPI_login_CL"
+		},
+		{
+			"metricName": "Total data received",
+			"legend": "GWorkspace_ReportsAPI_mobile_CL",
+			"baseQuery": "GWorkspace_ReportsAPI_mobile_CL"
+		},
+		{
+			"metricName": "Total data received",
+			"legend": "GWorkspace_ReportsAPI_token_CL",
+			"baseQuery": "GWorkspace_ReportsAPI_token_CL"
+		},
+		{
+			"metricName": "Total data received",
+			"legend": "GWorkspace_ReportsAPI_user_accounts_CL",
+			"baseQuery": "GWorkspace_ReportsAPI_user_accounts_CL"
+		}
+	],
+	"sampleQueries": [{
+		"description": "Last Google Workspace Activity Events",
+		"query": "GWorkspaceActivityReports\n | sort by TimeGenerated desc"
+	}],
+	"dataTypes": [{
+			"name": "GWorkspace_ReportsAPI_admin_CL",
+			"lastDataReceivedQuery": "GWorkspace_ReportsAPI_admin_CL\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+		},
+		{
+			"name": "GWorkspace_ReportsAPI_calendar_CL",
+			"lastDataReceivedQuery": "GWorkspace_ReportsAPI_calendar_CL\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+		},
+		{
+			"name": "GWorkspace_ReportsAPI_drive_CL",
+			"lastDataReceivedQuery": "GWorkspace_ReportsAPI_drive_CL\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+		},
+		{
+			"name": "GWorkspace_ReportsAPI_login_CL",
+			"lastDataReceivedQuery": "GWorkspace_ReportsAPI_login_CL\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+		},
+		{
+			"name": "GWorkspace_ReportsAPI_mobile_CL",
+			"lastDataReceivedQuery": "GWorkspace_ReportsAPI_mobile_CL\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+		},
+		{
+			"name": "GWorkspace_ReportsAPI_token_CL",
+			"lastDataReceivedQuery": "GWorkspace_ReportsAPI_token_CL\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+		},
+		{
+			"name": "GWorkspace_ReportsAPI_user_accounts_CL",
+			"lastDataReceivedQuery": "GWorkspace_ReportsAPI_user_accounts_CL\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+		}
+	],
+	"connectivityCriterias": [{
+			"type": "IsConnectedQuery",
+			"value": [
+				"GWorkspace_ReportsAPI_admin_CL\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
```

Comment:
Can you please have all the connectivity queries as a single entity with comma separated queries? Please refer another connector file here for sample: https://github.com/Azure/Azure-Sentinel/blob/3bbe94da22e1f55b75bf1fced4442903a9f7669c/DataConnectors/VMware%20Carbon%20Black/VMwareCarbonBlack_API_FunctionApp.json

Reply:
Fixed